sql/scan.trace

```diff
@@ -0,0 +1,12 @@
+statement ok
+CREATE TABLE t (k INT PRIMARY KEY)
+
+statement ok
+INSERT INTO t VALUES (1)
+
+traceon trace.out
```

Comment:
Oops. No. How did that sneak in there? I don't think this needs another `make check`. I'll remove.